package_managers/bootstrap_image.bzl

```diff
@@ -127,10 +127,11 @@ Args:
   package_manager_genrator: A target which generates a script using
        package management tool e.g apt-get, dpkg to downloads packages.
   store: A target to store the downloaded packages or retrieve previously downloaded packages.
-  additional_repos: list of additional debian package repos to use, in sources.list format
+  additional_repos: list of additional debian package repos to use, in sources.list format.
+  post_installation_script: extra commands to run after package installation.
```

Comment:
Thanks! Just one nit: could you rename this to something like "installation_cleanup_commands"? I understand your use-case and why it needs to be run here instead of as another layer, but naming it something generic like this could encourage it's misuse.

Reply:
Thanks! Done.